package.json

```diff
@@ -10,14 +10,36 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
```

Comment:
Would it be possible to add a `test` script that runs unit and e2e tests at once?